### ISSUE TITLE
vscode with rust-analyzer cannot work for vmm/sandbox crate

### ISSUE THREAD

**Issue by renzhengeek** [kuasar-io/kuasar#62]
Hi,

### What happened?

As the subject, in my dev setup, rust-analyzer works well for crates (i mean subproject with individual Cargo.toml), except vmm/sandbox crate. 

Does anyone see the same problem? My guess is that, it's related to the deeper directory struct of vmm/sandbox.

### What did you expect to happen?

code in vmm/sandbox subdir can works for rust-analyzer as other crates.

### How can we reproduce it (as minimally and precisely as possible)?

with vscode + rust-analyzer, try goto definition.

To make RA know all crates, add workspaces under root dir, ref to:
https://www.reddit.com/r/rust/comments/ikzbdj/vscode_multiple_project_roots_rustanalyzer_clippy/

Cargo.toml in root dir:
```
[workspace]

members = [
    "vmm/common",
    "vmm/sandbox",
    "vmm/task",
    "shim",
    "quark",
]

```

**Comment by Vanient** [comment]
Have you added any default feature in vmm/sandbox Cargo.toml, include qemu/stratovirt/cloud_hypervisor yet. Just like
```
[features]
default = ["qemu", "stratovirt", "cloud_hypervisor"]    // add in this line, add them all or just choose one or two
qemu = ["qapi/qmp", "qapi/async-tokio-all", "unshare", "os_pipe"]
cloud_hypervisor = ["api_client"]
stratovirt = ["qapi/qmp", "qapi/async-tokio-all", "unshare", "os_pipe"]
```

But this maybe no longer needed after #54 is solved

**Comment by renzhengeek** [comment]
yes, i also just figured out the same solution. thx!
